ctwrap/simulation.py

```diff
@@ -345,54 +348,54 @@ def _parse_output(dct: Dict[str, Any],
            fpath (string, optional): output path (overrides yaml)
 
         Returns:
-            Dictionary containng output information
+            Dictionary containing output information
         """
+        out = None
 
-        if dct is None:
-            dct = {}
+        if dct is not None:
```

Comment:
Not sure that this is the simplest change. You could simply do
```Python
if dct is None:
    return None
```
And leave the rest.